tests/e2e/actor_activation/actor_activation_test.go

```diff
@@ -101,7 +102,6 @@ func TestActorActivation(t *testing.T) {
 	require.NoError(t, err)
 
 	t.Run("Actor deactivates due to timeout.", func(t *testing.T) {
-		// TODO: Use UUID in ActorId once dapr/dapr#895 is fixed.
 		actorId := "100"
```

Comment:
Please change here as well : guuid.New().String()

Reply:
This is actually intentional to have one that's a guid and one that's not.